src/app/competition/form.tsx

```diff
@@ -61,6 +62,7 @@ export default function CompetitionSubmission() {
         <label htmlFor="email">University of Warwick Email Address</label>
         <br></br>
         <input
+          className="text-center w-1/2"
```

Comment:
w-1/2 is fine on medium displays but huge on large displays and too small on a phone.

```suggestion
          className="w-96 max-w-full text-center"
```